src/contract_wrappers/tokens/__tests__/security_token_wrapper.test.ts

```diff
@@ -0,0 +1,2862 @@
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.transfersFrozen).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('getModulesByName', () => {
+    test('should call to getModulesByName', async () => {
+      const expectedResult = true;
+      const mockedParams = {
+        moduleName: ModuleName.generalPermissionManager,
+      };
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getModulesByName).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync(stringToBytes32(mockedParams.moduleName))).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.getModulesByName(mockedParams);
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.getModulesByName).once();
+      verify(mockedMethod.callAsync(stringToBytes32(mockedParams.moduleName))).once();
+    });
+  });
+
+  describe('getInvestors', () => {
+    test('should call to getInvestors', async () => {
+      const expectedResult = ['string', 'string'];
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getInvestors).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.getInvestors();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.getInvestors).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('getInvestorsAt', () => {
+    test('should call to getInvestorsAt', async () => {
+      const expectedResult = ['string', 'string'];
+      const mockedParams = {
+        checkpointId: 1,
+      };
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getInvestorsAt).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync(objectContaining(numberToBigNumber(mockedParams.checkpointId)))).thenResolve(
+        expectedResult,
+      );
+
+      // Real call
+      const result = await target.getInvestorsAt(mockedParams);
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.getInvestorsAt).once();
+      verify(mockedMethod.callAsync(objectContaining(numberToBigNumber(mockedParams.checkpointId)))).once();
+    });
+  });
+
+  describe('iterateInvestors', () => {
+    test('should call to iterateInvestors', async () => {
+      const expectedResult = ['string', 'string'];
+      const mockedParams = {
+        start: new Date(),
+        end: new Date(),
```

Comment:
iterateInvestors should not receive dates. They should be `number` instead. This is wrong on the wrapper, could you fix it there too?
iterateInvestors is a way to get all investors by pages, `start` and `end` are the indexes you want to get.